docs/install.rst

```diff
@@ -1,13 +1,53 @@
 =====================
-Quick Installation
+Get Certbot
 =====================
 
-If ``certbot`` (or ``letsencrypt``) is packaged for your Unix OS (visit
-certbot.eff.org_ to find out), you can install it
-from there, and run it by typing ``certbot`` (or ``letsencrypt``).  Because
-not all operating systems have packages yet, we provide a temporary solution
-via the ``certbot-auto`` wrapper script, which obtains some dependencies from
-your OS and puts others in a python virtual environment::
+.. _installation:
+
+Certbot is packaged for many common operating systems and web servers. Check whether
+``certbot`` (or ``letsencrypt``) is packaged for your web server's OS by visiting
+certbot.eff.org_, where you will also find the correct installation instructions for
+your system.
+
+.. Note:: Unless you have very specific requirements, we kindly suggest that you use the Certbot packages provided by your package manager (see certbot.eff.org_). If such packages are not available, we recommend using ``certbot-auto``, which automates the process of installing Certbot on your system.
+
+.. _certbot.eff.org: https://certbot.eff.org
+
+.. _certbot-auto: https://certbot.eff.org/docs/install.html#certbot-auto
```

Comment:
This line causes errors when building the documentation because `_certbot-auto` is also defined below on line 45. The definition there is arguably more accurate and easier to maintain, so I think this line should be deleted.